pkgs/development/ocaml-modules/resto/directory.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, buildDunePackage, resto, resto-json, lwt }:
+
+buildDunePackage {
+  pname = "resto-directory";
+  inherit (resto) src version meta useDune2 doCheck;
+
+  propagatedBuildInputs = [
+    resto
+    resto-json
```

Comment:
Is `resto-json` useful?

Reply:
What do you mean? It's a dependency right?

Reply:
I don’t think so: https://gitlab.com/nomadic-labs/resto/-/blob/v0.6.1/src/dune#L64

Reply:
Thanks, I was picking dependencies from .opam files. I have now aligned all dependencies with the dune file instead.